### DISCUSSION TITLE
[14->15] Can't load migration module

### DISCUSSION THREAD

**Discussion by GruberViktor** [OCA/OpenUpgrade#3691] [Q&A] [answered]
Hi All,

I am trying to upgrade, but I am getting this error:

```
CRITICAL ? odoo.modules.module: Couldn't load module openupgrade_framework 
CRITICAL ? odoo.modules.module: type object 'NameManager' has no attribute 'check' 
ERROR ? odoo.service.server: Failed to load server-wide module `openupgrade_framework`. 
```
I downloaded the 15.0 branch of OpenUpgrade.

This is the command I am using in order to upgrade: `python3 odoo-bin -d odoo --addons-path /home/viktor/odoo/addons/ -r odoo -D /home/viktor/.local/share/Odoo/ --update all --stop-after-init --load=base,web,openupgrade_framework` 

Is this the way to upgrade the database? The instructions are a bit vague to me.

**Comment by GruberViktor** [comment, marked as answer]
Ah, now I get it. The upgrade must be run by starting 15.0. I always started it with 14.0. It is working now.

**Comment by Jny-z** [comment]
Hi, can you elaborate how this process done.
I have try using

- Create a db in odoo14 and take the bakeup  of the db
- then i move to odoo 15 environment there using the restore database i have insert the backup-ed 14 db to odoo15 environment.
- Now i use this db to run the migration process.

Just tell me is this the correct flow.

Thanks is Advance!
